src/components/Inputfield/ImageInput.jsx

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components';
+
+const ImageUpload = ({setValue}) => {
+	return (
+		<InputContainer>
+			<div>
+				<label htmlFor="image">Choose File</label>
+				<input type="file" id="image" hidden />
+				<span>Upload Eventpicture</span>
+			</div>
+		</InputContainer>
+	);
+};
+
+export default ImageUpload;
+
+const InputContainer = styled.div`
+	position: relative;
+	width: 16em;
+
+	label {
+		margin-top: 1rem;
+		padding: 0.5rem;
+		transition: 0.5s;
+		border-radius: 0.3rem;
+		color: rgba(255, 255, 255, 0);
+		font-family: sans-serif;
+		font-size: 0.8em;
+		text-transform: uppercase;
+		cursor: pointer;
+
+		&:hover {
+			color: white;
+		}
+	}
+
+	div {
```

Comment:
Perhaps you will find a more semantically descriptive element 🤟

Reply:
Done